state.json

```diff
@@ -227,7 +227,7 @@
                 "name": "sfo",
                 "random": true,
                 "commands": [
-                    "gslide id=1ZtS6XVAqrj2pQDPINdXnSVB3_AH1v4wQ_NGwtbD8QPw comment=\"SFO Zoom Transition\""
+                    "gslide id=1VQvRzDCwKOuIYdluBnAYLqN_iMKSKjaz7EaFivkIUTs comment=\"SFO Zoom Transition\""
```

Comment:
It looks like this slide deck isn't publically viewable, so Corsica won't be able to show the images. Can you fix that?

Reply:
Both decks are now publicly viewable.  I had published to the web, but they were still locked in my drive.  Fixed